api-reference/upload-file-api/secure-client-side-file-upload.md

```diff
@@ -24,102 +22,15 @@ A file can have a maximum of 100 versions.
-| <p><strong>customMetadata</strong></p><p>optional</p>                                         |  Stringified JSON key-value data to be associated with the asset. Checkout `overwriteCustomMetadata` parameter to understand default behaviour. Before setting any custom metadata on an asset you have to create the field using [custom metadata fields API](../custom-metadata-fields-api/).                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 |
-| <p><strong>transformation</strong></p><p>optional</p>                                       | Stringified JSON object with the properties: <br><ul><li> `pre` - Accepts a "string" containing a valid transformation. Used for requesting a pre-transformation for an image or a video file.</li> <li>`post` - Accepts an array of objects with the following properties: <ul><li>`type`: Either `transformation`, `gif-to-video`, `thumbnail` or `abs`</li><li>`value`: A "string" corresponding to the required transformation.<br/>Required if `type` is `transformation` or `abs`.<br/>Optional if `type` is `gif-to-video` or `thumbnail`.</li><li>`protocol`: Either `hls` or `dash`. Only applicable if `type` is `abs`.<br/>Read more about [Adaptive bitrate streaming (ABS)](../../features/video-transformation/adaptive-bitrate-streaming.md).</li></ul> Used for requesting post-transformations for an image or a video file.</li></ul>                           |
-
-## Response code and structure (JSON)
+The request structure is the same as the [server-side file upload API](./server-side-file-upload.md#request-structure-multipartform-data), except that to implement secure client-side upload, the application needs to pass a `token`, as explained below.
```

Comment:
"implement secure client-side upload" wording seems wrong.
"The application needs to send" - be more clear, similar to client side upload.

Reply:
done